_data/current.yml

```diff
@@ -1,4 +1,11 @@
-- name: The Premier Online Conference For Software Testing
+- name: Elevating Quality: Expert Insights on QA & Software Testing
+  location: LinkedIn Live Event
+  dates: "June 06, 2024"
```

Comment:
@naresh665 this is in the wrong location. Please move the whole conference block down so that its date matches